src-java/kilda-persistence-api/src/main/java/org/openkilda/persistence/repositories/LagLogicalPortRepository.java

```diff
@@ -29,4 +32,6 @@ public interface LagLogicalPortRepository extends Repository<LagLogicalPort> {
     Optional<LagLogicalPort> findBySwitchIdAndPortNumber(SwitchId switchId, int portNumber);
 
     Optional<Integer> findUnassignedPortInRange(SwitchId switchId, int portFirst, int portLast);
+
+    Map<SwitchId, List<LagLogicalPort>> findBySwitchIds(Set<SwitchId> switchIds);
```

Comment:
Is it really required? why caller can't use `findBySwitchId(SwitchId)` method by itself?